app/DashBoard/Presenters/templates/Blocks/favicons.latte

```diff
@@ -0,0 +1,8 @@
+{block favicon}
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
```

Comment:
Nejde o overkill, ale o zajištění, že to bude vždy fungovat. To, že tam není adresář je jen shoda náhod, stačí upravit router, přidat vícejazyčnost s monitoring.cz/cs/ a bude potřeba to řešit.